server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanExtension.java

```diff
@@ -81,9 +81,6 @@ public void initialize(ExtensionContext context) {
 
         subsystem.registerSubsystemModel(new InfinispanSubsystemRootResource(resolvePathHandler, context.isRuntimeOnlyRegistrationValid()));
         subsystem.registerXMLElementWriter(new InfinispanSubsystemXMLWriter());
-        if (context.isRegisterTransformers()) {
```

Comment:
I don't understand much of the server parser, but this changes does not make sense to me in this context... except if the transformers are only used in the old parsers (<1.4)